pkg/security/podsecuritypolicy/sysctl/mustmatchpatterns.go

```diff
@@ -35,6 +35,7 @@ func SafeSysctlWhitelist() []string {
 		"net.ipv4.ip_local_port_range",
 		"net.ipv4.tcp_syncookies",
 		"net.ipv4.ping_group_range",
+		"net.ipv4.ip_unprivileged_port_start",
```

Comment:
I'm surprised to see this... bad rebase?

Reply:
Yes, give me a moment to fix this